src/TLuaInterpreter.cpp

```diff
@@ -7925,6 +7925,50 @@ int TLuaInterpreter::exists(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#existsID
+int TLuaInterpreter::existsID(lua_State* L)
+{
+    auto id = getVerifiedInt(L, __func__, 1, "ID");
+    if (id < 0) {
+        // Must be zero or more but doesn't seem to be:
+        return warnArgumentValue(L, __func__, "item ID as %d does not seem to be a positive integer", id);
+    }
+    auto type = getVerifiedString(L, __func__, 2, "type").toLower();
+    Host& host = getHostFromLua(L);
+
+    if (!type.compare(QLatin1String("timer"), Qt::CaseInsensitive)) {
+        auto pT = host.getTimerUnit()->getTimer(id);
+        lua_pushboolean(L, static_cast<bool>(pT));
+        return 1;
+    }
+    if (!type.compare(QLatin1String("trigger"), Qt::CaseInsensitive)) {
+        auto pT = host.getTriggerUnit()->getTrigger(id);
+        lua_pushboolean(L, static_cast<bool>(pT));
+        return 1;
+    }
+    if (!type.compare(QLatin1String("alias"), Qt::CaseInsensitive)) {
+        auto pT = host.getAliasUnit()->getAlias(id);
+        lua_pushboolean(L, static_cast<bool>(pT));
+        return 1;
+    }
+    if (!type.compare(QLatin1String("keybind"), Qt::CaseInsensitive)) {
+        auto pT = host.getKeyUnit()->getKey(id);
+        lua_pushboolean(L, static_cast<bool>(pT));
+        return 1;
+    }
+    if (!type.compare(QLatin1String("button"), Qt::CaseInsensitive)) {
+        auto pT = host.getActionUnit()->getAction(id);
+        lua_pushboolean(L, static_cast<bool>(pT));
+        return 1;
+    }
+    if (!type.compare(QLatin1String("script"), Qt::CaseInsensitive)) {
+        auto pT = host.getScriptUnit()->getScript(id);
+        lua_pushboolean(L, static_cast<bool>(pT));
+        return 1;
+    }
+    return warnArgumentValue(L, __func__, qsl("invalid item type '%1' given, it should be one (case insensitive) of: 'alias', 'button', 'script', 'keybind', 'timer' or 'trigger'").arg(type));
```

Comment:
Improve readability and sentence flow
```suggestion
    return warnArgumentValue(L, __func__, qsl("invalid item type '%1' given, it should be one of: 'alias', 'button', 'script', 'keybind', 'timer' or 'trigger' (case insensitive)").arg(type));
```

Reply:
As above.